Refactored stringToB.rb

tom-lord opened the pull request:
Simplified syntax to improve readability

tom-lord committed:
Refactored stringToB.rb

Simplified syntax to improve readability

tom-lord committed:
Further refactor

Removed the "magic string" of BF characters

reynoldscem commented:
Cannot merge with the removal of the magic string unfortunately, will merge if another is submitted with the string in tact as I consider it a bit of fun.

tom-lord commented:
Meh... I was going for clean code over black magic, but fine - I'll revert
that second commit ;)
On 28 Jul 2015 01:55, "reynoldscem" notifications@github.com wrote:

> Cannot merge with the removal of the magic string unfortunately, will
> merge if another is submitted with the string in tact as I consider it a
> bit of fun.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/reynoldscem/brainrub/pull/1#issuecomment-125397483.

tom-lord committed:
:tada: Re-added black magic

++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++.++++++++++++++++++++.+.++++++++++.-----------------------------------------------------------------------------------.+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++.++++++++++.-----------------------------------------------------------------------------------.++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++.-----.+++++.-------------------.+++++++++++..+++++++++++++.-----------------------------------------------------------------------------------------.+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++.------------.++++++++.+++++.++++++.-------------------.++++++++.+++++.-------------.+.++++++++++.-------.-------------------------------------------------------...--------------.+++++++++++++++++++++++++++.------------------.-------------------------------.
